accepted/vulnerability-response-runbook.md

```diff
@@ -0,0 +1,203 @@
+# Bytecode Alliance Vulnerability Response Runbook
+
+## Select an Incident Manager
+
+Pick an individual who will track that the the entire process in this document isfollowed.
+By default, the person inside Bytecode Alliance who discovers or learns of the
+vulnerability first has this duty, until they explicitly hand if off to
+someone else.
+
+## Identifying the Vulnerability
+
+Figure out what bytecode alliance repository it lives in.
+
+The vulnerability probably lives in one single repository in the Bytecode
+Alliance ecosystem. If fixing it requires coordinated work across two
+repositories, this guide may not work.
```

Comment:
What should the reader do then? Maybe email security@bytecodealliance.org or ping folks on Zulip?